libs/drape/metal/metal_texture.mm

```diff
@@ -79,9 +97,23 @@ MTLPixelFormat UnpackFormat(TextureFormat format)
 {
   UNUSED_VALUE(context);
   CHECK(m_isMutable, ("Upload data is avaivable only for mutable textures."));
+  UploadDataImpl(x, y, width, height, 0, data);
```

Comment:
nit: Call UploadData(layer = 0) without UploadDataImpl if this class MetalTexture is final?